documentation/man/features/report.rst

```diff
@@ -55,6 +55,12 @@ OPTIONS
 \--year[=<year>]:
   Exhibits the current year summary if the user does not specify *<year>*.
 
+\--all:
+  Display all information for the current date.
+
+\--pomodoro:
+  Display current date pomodoro report.
```

Comment:
As I mentioned in #506 add/change this to clarify that the date and range can be set with the date options.